submissions/OlgaBarnikova/Popup/index.html

```diff
@@ -0,0 +1,157 @@
+                                <div>Promo</div>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="popup_box_img">
+                              <img src="popup-icons/tanslation.png" alt="" 
+                              class="popup_img" />
+                              <div>Translation</div>
+                            </a>
+                         </li>
+                         <li>
+                            <a href="#" class="popup_box_img">
+                              <img src="popup-icons/videos.png" alt="" 
+                              class="popup_img" />
+                              <div>Videos</div>
+                            </a>                            
+                         </li>   
+                    </ul>
+                    <ul class="popup_more">
+                        <li>
+                        <input id="popup-more" type="checkbox" class="popup_more_input">
+                        <label for="popup-more" class="popup_more_icon">More</label>
+                        <div class="popup_list_more">
+                            <ul class="popup_list">
+                                <li>
+                                    <a href="#" class="popup_box_img">
+                                        <img src="popup-icons/contacts.png" alt="" 
+                                        class="popup_img">
+                                        <div>Contacts</div>
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="#" class="popup_box_img">
+                                      <img src="popup-icons/photos.png" alt="" 
+                                      class="popup_img" />
+                                      <div>Photos</div>
+                                    </a>
+                                 </li>
+                                 <li>
+                                    <a href="#" class="popup_box_img">
+                                      <img src="popup-icons/classroom.png" alt="" 
+                                      class="popup_img" />
+                                      <div>Classroom</div>
+                                    </a>                            
+                                 </li> 
+                            </ul>
+                        </div>
+                        </li>
+                    </ul> 
+                </div>
+
+            </li>
+            <li class="navigation_item">
+                <a href="#"><img src="popup-icons/favicon.png" alt="" 
+                    class="popup_img" /></a>
+            </li>
+            <li class="navigation_item">
+                <a href="#"><img src="popup-icons/bell.png" alt="" 
+                    class="popup_img" /></a>
+            </li>
+        </ul>
+    </nav>
+</body>
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)